packages/drivers/odsp-driver-definitions/src/errors.ts

```diff
@@ -46,6 +46,16 @@ export enum OdspErrorType {
     // SPO can occasionally return 403 for r/w operations on document when there is a fail over to another data center.
     // So to preserve integrity of the data, the data becomes readonly.
     serviceReadOnly = "serviceReadOnly",
+
+    /**
+     * Invalid item id (during creation of Fluid container on existing file)
+     */
+    invalidItemIdError = "invalidItemIdError",
```

Comment:
i don't think these error fit with the kinds of errors we usually put in this enum. i think in both cases you can just throw a usage error as they point to an incorrect api call, and pass this as the details as a string

Reply:
Thanks. I'll remove the changes here and use UsageError instead.